Resources/Prototypes/_LostParadise/Loadouts/Jobs/justice.yml

```
@@ -0,0 +1,36 @@
+      jobs:
+        - ChiefJustice
+        - Clerk
+  items:
+    - ClothingOuterRobesJudge
+
+- type: loadout
+  id: LoadoutLPPWeaponRevolverJudge
+  category: JobsCommand
+  cost: 6
+  exclusive: true
+  requirements: 
```

yamllint

[error] trailing spaces

(trailing-spaces)